Formula/geant4.rb

```diff
@@ -0,0 +1,101 @@
+  version "10.4.2"
+  sha256 "2cac09e799f2eb609a7f1e4082d3d9d3d4d9e1930a8c4f9ecdad72aad35cdf10"
+
+  option "with-g3tog4", "Use G3toG4 Library"
+  option "with-usolids", "Use USolids (experimental)"
+  option "without-multithreaded", "Build without multithreading support"
+
+  depends_on "cmake"
+  depends_on :x11
+  depends_on "xerces-c" => :recommended
+  depends_on "qt" => :optional
+
+  resource "G4NDL" do
+    url "http://cern.ch/geant4-data/datasets/G4NDL.4.5.tar.gz"
+    sha256 "cba928a520a788f2bc8229c7ef57f83d0934bb0c6a18c31ef05ef4865edcdf8e"
+  end
+
+  resource "G4EMLOW" do
+    url "http://cern.ch/geant4-data/datasets/G4EMLOW.7.3.tar.gz"
+    sha256 "583aa7f34f67b09db7d566f904c54b21e95a9ac05b60e2bfb794efb569dba14e"
+  end
+
+  resource "PhotonEvaporation" do
+    url "http://cern.ch/geant4-data/datasets/G4PhotonEvaporation.5.2.tar.gz"
+    sha256 "83607f8d36827b2a7fca19c9c336caffbebf61a359d0ef7cee44a8bcf3fc2d1f"
+  end
+
+  resource "RadioactiveDecay" do
+    url "http://cern.ch/geant4-data/datasets/G4RadioactiveDecay.5.2.tar.gz"
+    sha256 "99c038d89d70281316be15c3c98a66c5d0ca01ef575127b6a094063003e2af5d"
+  end
+
+  resource "G4SAIDDATA" do
+    url "http://cern.ch/geant4-data/datasets/G4SAIDDATA.1.1.tar.gz"
+    sha256 "a38cd9a83db62311922850fe609ecd250d36adf264a88e88c82ba82b7da0ed7f"
+  end
+
+  resource "G4NEUTRONXS" do
+    url "http://cern.ch/geant4-data/datasets/G4NEUTRONXS.1.4.tar.gz"
+    sha256 "57b38868d7eb060ddd65b26283402d4f161db76ed2169437c266105cca73a8fd"
+  end
+
+  resource "G4ABLA" do
+    url "http://cern.ch/geant4-data/datasets/G4ABLA.3.1.tar.gz"
+    sha256 "7698b052b58bf1b9886beacdbd6af607adc1e099fc730ab6b21cf7f090c027ed"
+  end
+
+  resource "G4PII" do
+    url "http://cern.ch/geant4-data/datasets/G4PII.1.3.tar.gz"
+    sha256 "6225ad902675f4381c98c6ba25fc5a06ce87549aa979634d3d03491d6616e926"
+  end
+
+  resource "G4ENSDFSTATE" do
+    url "http://cern.ch/geant4-data/datasets/G4ENSDFSTATE.2.2.tar.gz"
+    sha256 "dd7e27ef62070734a4a709601f5b3bada6641b111eb7069344e4f99a01d6e0a6"
+  end
+
+  resource "RealSurface" do
+    url "http://cern.ch/geant4-data/datasets/G4RealSurface.2.1.1.tar.gz"
+    sha256 "90481ff97a7c3fa792b7a2a21c9ed80a40e6be386e581a39950c844b2dd06f50"
+  end
+
+  def install
+    mkdir "geant-build" do
+      args = %w[
+        ../
+        -DGEANT4_USE_OPENGL_X11=ON
+        -DGEANT4_USE_RAYTRACER_X11=ON
+      ]
+
+      args << "-DGEANT4_USE_QT=ON" if build.with? "qt"
+      args << "-DGEANT4_USE_G3TOG4=ON" if build.with? "g3tog4"
+      args << "-DGEANT4_USE_GDML=ON" if build.with? "xerces-c"
+      args << "-DGEANT4_USE_USOLIDS=ON" if build.with? "usolids"
+      args << "-DGEANT4_BUILD_MULTITHREADED=ON" if build.with? "multithreaded"
+      args.concat(std_cmake_args)
+      system "cmake", *args
+      system "make", "install"
+    end
+  end
+
+  def post_install
+    resources.each do |r|
+      (share/"Geant4-#{version}/data/#{r.name}#{r.version}").install r
```

Comment:
Why is this in postinstall and not in install?

Reply:
I believe in post install they don't become part of the bottle (one or two of them are large), and you can also rerun the post-install without rebuilding (that's how I was working on developing it). If there's a reason to move it, I can.

Reply:
Move to install, please.

Reply:
Also: are these datasets required to make the software useful? I'm not sure why a simulation toolkit would require large datasets, except as examples or tests